data/json/recipes/food/dairy_products.json

```diff
@@ -29,7 +29,7 @@
     "time": "20 m",
     "book_learn": [ [ "dairy_book", 3 ] ],
     "//": "Book Things to do with milk.  Add curdled milk and cheese recipes to the book.  Also consider adding to brewing json Airag from this book.",
-    "tools": [ [ [ "jar_glass_sealed", 1 ] ] ],
+    "tools": [ [ [ "jar_glass_sealed", -1 ] ] ],
```

Comment:
This recipe should have a byproduct of four buttermilk as well.

Reply:
The original recipe took this into account already. Line 23.

Reply:
I must have been blind when I read the recipe looking for that.   :)

Reply:
No problem.